src/routers/handlers/is_business_registered/index.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response } from "express";
+import { createAndLogError, logger } from "../../../utils/logger";
+import { getIsBusinessRegisteredFromExtraData, setExtraDataIsBusinessRegistered } from "../../../utils/session";
+import { BaseViewData, GenericHandler, Redirect, ViewModel } from "../generic";
+import { PrefixedUrls } from "../../../constants";
+import { addLangToUrl, getLocalesField, selectLang } from "../../../utils/localise";
+
+interface IsBusinessRegisteredViewData extends BaseViewData {
+    isBusinessRegistered?: boolean;
+}
+
+const isBusinessRegisteredKey = "is-business-registered";
+
+export class IsBusinessRegisteredHandler extends GenericHandler<IsBusinessRegisteredViewData> {
+    private static readonly templatePath = "router_views/is_business_registered/is_business_registered";
+
+    /**
+     * Gets the viewdata from the request
+     * @param req : Request
+     * @returns: IsBusinessRegisteredViewData
+     */
+    public getViewData(req: Request): IsBusinessRegisteredViewData {
+        const baseViewData = super.getViewData(req);
+        return {
+            ...baseViewData,
+            title: "Is the business you work for registered with Companies House? – File package accounts with Companies House – GOV.UK",
+            backURL: null,
```

Comment:
Now this is the first page of the service, it should have the back url to the gds page, how the enter your details had